dot/rpc/subscription/listeners_test.go

```diff
@@ -344,7 +344,7 @@ func TestRuntimeChannelListener_Listen(t *testing.T) {
 
 	expectedInitialVersion := modules.StateRuntimeVersionResponse{
 		SpecName: "mock-spec",
-		Apis:     modules.ConvertAPIs(nil),
+		Apis:     []interface{}{},
```

Comment:
Does this still need to be an interface or can we be more explicit with the type?

Reply:
A bit out of scope here, I'm just replacing the return from `modules.ConvertAPIs(nil)`.

Anyway, yes, we need this, since we use it with `[]interface{}{[]interface{}{"0x0102030405060708", uint32(99)}}}` so we kinda need interfaces to mix types in the same slice.